File: Unidad 4/C#/Prog_5_Uni_4.cs
using System;

namespace Prog_5_Uni_4 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string[] frutas = { "Manzana", "Fresa", "Cereza", "Frambuesa", "Granada" };


            string[] frutas_AlReves = new string[frutas.Length];


            for (int i = 0; i < frutas.Length; i++)
            {
                frutas_AlReves[i] = frutas[frutas.Length - 1 - i];
            }


            Console.WriteLine("Arreglo Original:");
            ImprimirArreglo(frutas);


            Console.WriteLine("\nArreglo al Revés:");
            ImprimirArreglo(frutas_AlReves);
        }

        static void ImprimirArreglo(string[] array)
        {
            foreach (string elemento in array)
            {
                Console.Write($"{elemento} ");
            }
            Console.WriteLine();
        }
    }
}

File: Unidad 4/Java/Prog_5_Uni_4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.prog_5_uni_4;

/**
 *
 * @author USUARIO LENOVO
 */
public class Prog_5_Uni_4 {

    public static void main(String[] args) {
        String[] frutas = { "Manzana", "Fresa", "Cereza", "Frambuesa", "Granada" };
        String[] frutasAlReves = new String[frutas.length];

        for (int i = 0; i < frutas.length; i++) {
            frutasAlReves[i] = frutas[frutas.length - 1 - i];
        }

        System.out.println("Arreglo Original:");
        imprimirArreglo(frutas);

        System.out.println("\nArreglo al Revés:");
        imprimirArreglo(frutasAlReves);
    }

    static void imprimirArreglo(String[] array) {
        for (String elemento : array) {
            System.out.print(elemento + " ");
        }
        System.out.println();
    }
}
